hera_qm/version.py

```diff
@@ -1,36 +1,55 @@
 # -*- coding: utf-8 -*-
-# Copyright (c) 2018 the HERA Project
+# Copyright (c) 2019 the HERA Project
 # Licensed under the MIT License
 
 import os
 import subprocess
 import json
+import six
+
+
+hera_qm_dir = os.path.dirname(os.path.realpath(__file__))
```

Comment:
Most of the changes in this file are getting the str/bytes stuff to work in py3. I don't understand it... mostly taken from pyuvdata.